package.json

```diff
@@ -109,9 +113,10 @@
     "@types/node": "8.0.28",
     "bl": "1.2.1",
     "body-parser": "1.18.0",
-    "chromedriver": "2.32.3",
     "brolog": "1.2.6",
+    "chromedriver": "2.32.3",
     "express": "4.15.2",
+    "markdown-include": "^0.4.3",
```

Comment:
I'd like to do not add ` markdown-include` dependence because it seems the job could be done without it.

Reply:
nop, this dependence is necessary. Or I have to write something like this.